go-controller/pkg/ovn/namespace.go

```diff
@@ -330,26 +330,38 @@ func (oc *DefaultNetworkController) getAllHostNamespaceAddresses() []net.IP {
 				// skip hybrid overlay nodes
 				continue
 			}
-			hostSubnets, err := util.ParseNodeHostSubnetAnnotation(node, types.DefaultNetworkName)
+			hostNetworkIPs, err := oc.getHostNamespaceAddressesForNode(node)
 			if err != nil {
-				klog.Warningf("Error parsing host subnet annotation for node %s (%v)",
-					node.Name, err)
-			}
-			for _, hostSubnet := range hostSubnets {
-				mgmtIfAddr := util.GetNodeManagementIfAddr(hostSubnet)
-				ips = append(ips, mgmtIfAddr.IP)
-			}
-			// for shared gateway mode we will use LRP IPs to SNAT host network traffic
-			// so add these to the address set.
-			lrpIPs, err := util.ParseNodeGatewayRouterLRPAddrs(node)
-			if err != nil {
-				klog.Errorf("Failed to get join switch port IP address for node %s: %v", node.Name, err)
-			}
-
-			for _, lrpIP := range lrpIPs {
-				ips = append(ips, lrpIP.IP)
+				klog.V(5).Infof("Error parsing annotation for node %s: %v", node.Name, err)
```

Comment:
This should not be Infof level right?
This is digressing from previously how it used to be where we are printing them at Warning,Error levels? and Why is this Level5 ?

Reply:
getAllHostNamespaceAddresses does not return the error to calling function, so i thought its good to print verbose message in case getHostNamespaceAddressesForNode fails. I should probably make it klog.Errof("") instead of klog.V(5).Infof and stop printing errors/warnings at getHostNamespaceAddressesForNode.